example.js

```diff
@@ -1,5 +1,5 @@
 var remark = require('remark');
-var wrap = require('./dist/index.js');
+var wrap = require('./src/index.js');
```

Comment:
This should probably still point to the babel-ified `dist` version
https://github.com/remarkjs/remark-word-wrap/blob/84c4cbca9e026ff21f051f1683426606e9aceef4/package.json#L13

```suggestion
var wrap = require('./dist/index.js');
```

Reply:
I trust you more than I trust me. Is there a trick to running the tests in this case?